web/templates/compose_banner/messages_fading_banner.hbs

```diff
@@ -0,0 +1,10 @@
+{{#> compose_banner }}
+    <p class="banner_message">
+        {{#tr}}
+        In Zulip, you can send messages to a conversation without going there.
+        You are composing a message to <z-link>{recipient}</z-link>.
+        Messages in other conversations are faded to avoid distraction.
+        {{#*inline "z-link"}}<a class="above_compose_banner_action_link" href="{{narrow_url}}">{{> @partial-block}}</a>{{/inline}}
+        {{/tr}}
```

Comment:
Seeing this in action, it feels very wordy, and I don't like how the recipient is quite likely to be line-wrapped. I'll post a proposal in #design.

Reply:
Updated the PR with no recipients. Though we are still finalizing the exact text.